api/db/backup.js

```diff
@@ -0,0 +1,47 @@
+const exec = require('child_process').exec;
+
+// Retrieve database connection parameters
+const getDbUrl = () => {
+  require('dotenv').config();
+  return process.env.DB_URL;
+};
+
+const createBackup = () => {
+  // Prepare database archive filename
+  currentDate = new Date();
+
+  let databaseDumpFilename =
```

Comment:
A few suggestions here:
- Use `const` instead of `let`
- Remove `database` from the name
- Use template literals/strings
- As the file will go inside `db/dumps`, maybe we can just name it `yearmonthdate.gz`
  - Please create the `db/dumps` folder with an empty file inside it called `.keep` (as it's done with the `uploads` folder)